cypress/e2e/reviewRequests.spec.ts

```diff
@@ -108,127 +157,510 @@ describe("Review Requests", () => {
+      cy.contains("In review").should("exist")
+      cy.contains("In review").click()
+      // Wait for dropdown menu to disappear
+      cy.wait(1000)
+
       // Assert
+      cy.contains("button", "Approved").should("not.be.visible")
+      cy.contains("button", "In review").should("be.visible")
     })
     it("should allow the requestor to close the review request", () => {
       // Arrange
+      const RR_INTERCEPTOR = "getReviewRequest"
+      cy.intercept("GET", "**/summary").as(RR_INTERCEPTOR)
+      const RR_CANCEL_INTERCEPTOR = "getReviewRequest"
+      const regex = /\/review\/\d+\//
+      cy.intercept("DELETE", regex).as(RR_INTERCEPTOR)
+      cy.setEmailSessionDefaults("Email admin")
+      cy.actAsEmailUser(E2E_EMAIL_ADMIN.email, "Email admin")
+      editUnlinkedPage(
+        "faq.md",
+        "some requestor close review request content",
+        E2E_EMAIL_TEST_SITE.repo
+      )
+      createReviewRequest(
+        MOCK_REVIEW_TITLE,
+        [E2E_EMAIL_ADMIN_2.email],
+        MOCK_REVIEW_DESCRIPTION
+      )
+      visitE2eEmailTestRepo()
+
       // Act
+      getViewReviewRequestButton().click()
+      cy.wait(`@${RR_INTERCEPTOR}`)
+      openReviewRequestStateDropdown("In review")
+      cy.contains("Cancel request").should("exist")
+      cy.contains("Cancel request").click()
+      cy.contains("Yes, cancel").click()
+      cy.wait(`@${RR_CANCEL_INTERCEPTOR}`)
+
       // Assert
+      cy.url().should("include", "/dashboard")
+      visitE2eEmailTestRepo()
+      getReviewRequestButton().should("exist")
     })
     it("should disallow users from viewing a closed review request", () => {
       // Arrange
-      // Act
-      // Assert
+      editUnlinkedPage(
+        "faq.md",
+        "some disallow view closed review request content",
+        E2E_EMAIL_TEST_SITE.repo
+      )
+      createReviewRequest(
+        MOCK_REVIEW_TITLE,
+        [E2E_EMAIL_ADMIN_2.email],
+        MOCK_REVIEW_DESCRIPTION
+      ).then((id) => {
+        closeReviewRequest(id)
+        // Act
+        cy.visit(`/sites/e2e-email-test-repo/review/${id}`)
+        // Assert
+        cy.contains(
+          "Please ensure that you have selected a valid review request."
+        )
+      })
     })
     it("should disallow users from viewing a merged review request", () => {
       // Arrange
-      // Act
-      // Assert
+      editUnlinkedPage(
+        "faq.md",
+        "some disallow view merged review request content",
+        E2E_EMAIL_TEST_SITE.repo
+      )
+      createReviewRequest(
+        MOCK_REVIEW_TITLE,
+        [E2E_EMAIL_ADMIN_2.email],
+        MOCK_REVIEW_DESCRIPTION
+      ).then((id) => {
+        cy.actAsEmailUser(E2E_EMAIL_ADMIN_2.email, "Email admin")
+        approveReviewRequest(id)
+        cy.actAsEmailUser(E2E_EMAIL_ADMIN.email, "Email admin")
+        mergeReviewRequest(id)
+        // Act
+        cy.visit(`/sites/e2e-email-test-repo/review/${id}`)
+        // Assert
+        cy.contains(
```

Comment:
same as above - wdyt about checking for redirection?

Reply:
734f4d22